src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayMiscellaneousOptions.cs

```diff
@@ -71,6 +71,6 @@ public enum SymbolDisplayMiscellaneousOptions
         /// <summary>
         /// Append '!' to non-nullable reference types.
```

Comment:
Do we care about this one?

Reply:
No, I think this is correct. That's how the spec describes such types:

> The non_nullable_reference_type in a nullable_reference_type must be a non-nullable reference type (class, interface, delegate or array), or a type parameter that is constrained to be a non-nullable reference type (through the class constraint, or a class other than object).